README.md

```diff
@@ -1,8 +1,76 @@
 # Dynamic DNS Update
 
-An application which updates the DNS records in your favourite providers, when your external/internet-facing IP address changes.
+An application which updates the DNS records in your DNS providers when your external/internet-facing IP address changes.
+
 Update checks can occur on a frequency specified in configuration.
 
+## Supported DNS providers
```

Comment:
May be worth adding support DNS Record Types as well as currently application only supports A Records but it needs to be specified in the config also.

Reply:
Good point. I've pushed an update.